products.yml

```diff
@@ -10,22 +10,22 @@ couchbase-server:
     release:  '6.5.2'
     stable:   '6.5.2'
   '6.6':
-    release:  '6.6.6-10576'
-    stable:   '6.6.6-10576'
+    release:  '6.6.6'
+    stable:   '6.6.6'
   '7.0':
     release:  '7.0.4'
     stable:   '7.0.4'
   '7.1':
-    release:  '7.1.6-3954'
-    stable:   '7.1.6-3954'
+    release:  '7.1.6'
+    stable:   '7.1.6'
```

Comment:
Cant update below 7.2 currently as the ec2 deployments wont work due to the centos7 AMI no longer existing. I think leave out of this patch for now, and if we want these EOL versions to not have buld no's, I can create some images for these if needs be.